pages/common/curl.md

```diff
@@ -3,34 +3,34 @@
 > Transfers data from or to a server.
 > Supports most protocols including HTTP, FTP, POP.
 
-- Download a URL to a file:
+- Download the contents of an URL to a file:
 
-`curl "{{URL}}" -o {{filename}}`
+`curl {{http://example.com}} -o {{filename}}`
 
-- Send form-encoded data:
+- Download a file saving it under the same name as indicated in the URL:
```

Comment:
I was thinking that when there is just one parameter in the example, there should be no need for a `t[e]xt`.